modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/IgniteH2Indexing.java

```diff
@@ -1530,6 +1543,9 @@ private void createSqlFunctions(String schema, Class<?>[] clss) throws IgniteChe
 
     /** {@inheritDoc} */
     @Override public void unregisterCache(CacheConfiguration<?, ?> ccfg) {
+        schema2space.remove(ccfg.getSqlSchema() == null ? '\"' + ccfg.getName() + '\"' : ccfg.getSqlSchema());
```

Comment:
We will do `String schema = schema(ccfg.getName());` on line 1549 why do not use schema name from there instead of calculating it. It is a third time I see schema name is calculated in the code. DRY!!!